src/controllers/midi/midicontroller.cpp

```diff
@@ -631,6 +631,12 @@ QJSValue MidiController::makeInputHandler(int status, int midino, const QJSValue
         return QJSValue();
     }
 
+    if (m_pMapping->getInputMappings().contains(status)) {
```

Comment:
This just check if an existing mapping exists. Should it perform a more thourough search to check if existing binding is an XML type?

Reply:
Before, all bindings are from XML right? So the question is regarding to allow a second js binding. Is that your point? Do we have a use case for it? I think in such cases you could just put both features onto one callback.

Reply:
I wonder why `midino` is not relevant here?

Reply:
Oh that's a mistake. Fixed.